### DISCUSSION TITLE
Proposed Scenario Order Change for Time Twister

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#22055] [Feature requests] [answered]
After conducting a poll, then personally vetting the scenarios in collaboration with others on the discord, I have here a suggested new scenario order for the Time Twister expansion pack:

Beginner Parks
Robin Hood
After the Asteroid
Prison Island
First Encounters

Challenging Parks
Schneider Cup
Stone Age
Cradle of Civilization
Rock 'n' Roll (Revival)
Future World

Expert Parks
Skyscrapers
Flower Power
Castle
Animatronic Film Set
Jurassic Safari

**Comment by J-PIE-314** [comment, marked as answer]
Changes to the order of Time Twisters scenarios were implemented in PR #22057.
